README.md

```diff
@@ -2265,6 +2272,47 @@ AccordionItem default expand
 </Accordion>
 
 ```
+### ListBlock
+
+Used to entails the presence of two distinct columns, separated by a divider with each column presenting a list of items in ordered or unordered.
+
+![listblock](docs/images/list-block.png)
+
+```
+<ListBlock slots="left, right" repeat="4" list="unorder" iconColor="#2ac3a2" icon="checkmark" variant="fullWidth" />
+
+500 free Document Transactions per month
+
+Volume and multi-product discounts
+
+Access to all 15+ PDF Services including PDF Extract, PDF Accessibility Auto-Tag API, and Document Generation
+
+Access to all 15+ PDF Services including PDF Extract, PDF Accessibility Auto-Tag API, and Document Generation
+
+Easy to sign up and create credentials in minutes
+
+Technical Support included (different tiers available)
+
+No credit card or commitment required
+
+Scalable for high volume needs.
+```
+
+Use `slots` to identify the markdown content:
+
+- `left`(required)
+
+- `right`(required)
+
+Use `repeat` to define how many code sections are part.
+
+Use `list` to define the order or unorder list. Defaults to `unorder`.
+
+Use `iconColor` to define the marker color. The preset is black.
+
+Use `icon` used to indicate individual items or elements within the list. If the list is in order by default it is `disc` else `number`. Values are `checkmark,disc, number`
```

Comment:
It seems like the default of the order list is number and not disc.  Also, order list seems to be not able to take in checkmark.  If this is by design, we may want to make sure the user knows about this.  For unorder list, the default is disc.  We may want to make this clear to the user as well.

Reply:
I removed the ordered/unordered list